### ISSUE TITLE
Fix `font-family-no-missing-generic-family-keyword` false positives for `math`, `emoji` and `fangsong`

### ISSUE THREAD

**Issue by otomad** [stylelint/stylelint#8390]
### What minimal example or steps are needed to reproduce the bug?

```css
.math {
    font-family: "Cambria Math", math;
}

.emoji {
    font-family: "Apple Color Emoji", "Segoe UI Emoji", "Noto Color Emoji", emoji;
}

.fangsong {
    font-family: "SimFang", "STFangsong", fangsong;
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
    "rules": {
        "font-family-no-missing-generic-family-keyword": true
    }
}
```

### How did you run Stylelint?

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIB0BbAhgFwBYAJgB0A7HEnAMwkIwFoy0UBLAGwE94d8QBhegIwCcGaHAFlMWTgBoc6bAG4iAXyJEkMFBABWDPEVLlKNOo1btOAQQAOlpjBxcITCPxwBRDdqkcQAZRgBzCDsAVQBJNw8GL04AOQgMCHtHZwitKJBpdTSFQmVCVTpCfwBnSn9dYlIKKlp6ZjZvHwYUADE0IujfABU2op8IL0KSspy8jJAyZhgY+jhEGAAPehsYJDBi4vHIQkn-BBACSpJOfgBXW2LOdkP9fU5qozrWakIIakYNhiLqfxhCGEEYFqJhY1AA1jAWAB3ZxQK44DBnGB6Uh5RRbSi7FrOWT7TSlQjjWCWTaIG7eYoYFi2JhfDDwzhMTAwSlSFGcSnUmC0mrbXbUSntKBofhwhDeJkYFn0kBKECKIA)

### Which Stylelint-related dependencies are you using?

```json
{
    "stylelint": "16.14.1"
}
```

### What did you expect to happen?

`math`, `emoji`, `fangsong` are generic family font keywords, they should be included in `font-family-no-missing-generic-family-keyword`.

[MDN Reference](https://developer.mozilla.org/docs/Web/CSS/font-family)

### What actually happened?

The following problems were reported:

```
example.css
2:18-32 ✖ Unexpected missing generic font family (font-family-no-missing-generic-family-keyword)
6:77-82 ✖ Unexpected missing generic font family (font-family-no-missing-generic-family-keyword)
10:41-49 ✖ Unexpected missing generic font family (font-family-no-missing-generic-family-keyword)
```

### Do you have a proposal to fix the bug?

https://github.com/stylelint/stylelint/blob/aed00cd75ecc3665e776f8d5d974c83d24212483/lib/reference/keywords.mjs#L14-L25
https://github.com/stylelint/stylelint/blob/aed00cd75ecc3665e776f8d5d974c83d24212483/lib/reference/keywords.cjs#L18-L29

Change to

```javascript
export const fontFamilyKeywords = uniteSets(basicKeywords, [ 
 	'serif', 
 	'sans-serif', 
 	'cursive', 
 	'fantasy', 
 	'monospace', 
 	'system-ui', 
 	'ui-serif', 
 	'ui-sans-serif', 
 	'ui-monospace', 
 	'ui-rounded', 
    'emoji',
    'math',
    'fangsong',
]); 
```

**Comment by Mouvedia** [comment]
I've labeled the issue as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.

There are [steps on how to fix a bug in a rule](https://stylelint.io/developer-guide/rules#fix-a-bug-in-a-rule) in the Developer guide.

**Comment by skyclouds2001** [comment]
I've fired https://github.com/mdn/data/pull/952 to fix it in upstream
